playwright/tests/integration/properties/layout.spec.ts

```diff
@@ -84,13 +84,16 @@ test('extract Dialog', async ({ page }) => {
   const { editor, layoutBlock } = await layout(page);
   await editor.createBlock('Input', layoutBlock.block);
   await layoutBlock.quickAction('Extract into own Ivy Component (E)');
-  await expect(page.getByRole('dialog')).toBeVisible();
-  const name = page.locator('.extract-dialog-name input');
-  const namespace = page.locator('.extract-dialog-namespace input');
-  const dataclass = page.locator('.extract-dialog-dataclass button');
-  await expect(name).toHaveValue('layout1');
-  await expect(namespace).toHaveValue('temp');
-  await expect(dataclass).toHaveText(/data/);
+  const dialog = page.getByRole('dialog');
+  await expect(dialog).toBeVisible();
+
+  const nameInput = dialog.getByRole('textbox', { name: 'Name', exact: true });
+  const namespaceInput = dialog.getByRole('textbox', { name: 'Namespace', exact: true });
+  const dataclassSelect = dialog.getByRole('combobox', { name: /data/i });
+
+  await expect(nameInput).toHaveValue('layout1');
+  await expect(namespaceInput).toHaveValue('temp');
+  await expect(dataclassSelect).toHaveText(/data/);
```

Comment:
if you only use the locator once, I would not split the locator from it's assertion. This makes only the code longer, and harder to read IMO.